apps/api/src/project/project.controller.ts

```diff
@@ -120,4 +124,32 @@ export class ProjectController {
       search
     )
   }
+
+  @Get(':projectSlug/export-secrets')
+  @RequiredApiKeyAuthorities(
+    Authority.READ_PROJECT,
+    Authority.READ_SECRET,
+    Authority.READ_VARIABLE,
+    Authority.READ_ENVIRONMENT
+  )
+  async exportProjectSecrets(
```

Comment:
`exportProjectConfigurations` would be a better name since we aim to export both variables and secrets. Same goes for the service layer